components/Preference.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Head from 'next/head';
+
+const Preference = (props) => {
+console.log('Preference being rendered')
+const pref = props.preference
+  return (
+    <div>
+      <main>
```

Comment:
\<main\> shouldn't be used here. Each page should have a maximum of 1 \<main\> component.